src/components/Integrations/BigQuery/ControlPanel/ControlPanel.tsx

```diff
@@ -0,0 +1,175 @@
+  const [saved, setSaved] = useState(false);
+
+  const onClickHandler = useCallback(
+    (isSaved: boolean) => {
+      if (isSaved) {
+        setMessage(isSaved);
+        setSaved(isSaved);
+      } else {
+        setMessage(isSaved);
+        setSaved(isSaved);
+      }
+    },
+    [setMessage, setSaved],
+  );
+
+  const onClickBtnHandler = useCallback(() => {
+    setMessage(true);
+    setSaved(false);
+  }, [setMessage, setSaved, values]);
+
+  const MessageMemo = useMemo(() => {
+    let messageText;
+    if (saved) {
+      messageText = <p>{i18('big_query.plus_message_saved')}</p>;
+    } else {
+      messageText = <p>{i18('big_query.plus_message_remember')}</p>;
+    }
+
+    if (message === true) {
+      return (
+        <div className="p-t-96" data-testid="message-info">
+          <div className={`dp-wrap-message dp-wrap-${saved ? 'success' : 'info'}`}>
+            <span className="dp-message-icon"></span>
+            <div className="dp-content-message">{messageText}</div>
+          </div>
+        </div>
+      );
+    }
+  }, [saved, message, i18]);
+
+  return (
+    <>
+      <header className="hero-banner">
+        <div className="dp-container">
+          <div className="dp-rowflex">
+            <div className="col-sm-12 col-md-12 col-lg-12">
+              <nav className="dp-breadcrumb">
+                <ul>
+                  <li>
+                    <a href={i18('big_query.url_legacy_doppler')}>
+                      {i18('big_query.plus_beginning')}
+                    </a>
+                  </li>
+                  <li>
+                    <a href={i18('big_query.url_legacy_control_panel')}>
+                      {i18('big_query.plus_configuration')}
+                    </a>
+                  </li>
+                  <li>
+                    <a href={i18('big_query.url_legacy_control_panel')}>
+                      {i18('big_query.plus_control_panel')}
+                    </a>
+                  </li>
+                  <li>{i18('big_query.plus_big_query')}</li>
+                </ul>
+              </nav>
+              <h2>{i18('big_query.plus_title')}</h2>
+            </div>
+            <div className="col-sm-7">
+              <FormattedMessageMarkdown id="big_query.plus_header_MD" />
+            </div>
+          </div>
+          <span className="arrow"></span>
+        </div>
+      </header>
+      <div className="dp-container">
+        <div className="dp-rowflex">
+          <div className="col-lg-5">
+            <div>
+              <span>{i18('big_query.plus_step_one')}</span>
+              <strong>
+                <FormattedMessageMarkdown id="big_query.plus_body_step_one_HD" />
+              </strong>
+              <p className="p-t-36">{i18('big_query.plus_step_one_paragraph')}</p>
+            </div>
+
+            <div className="dp-rowflex p-t-48">
+              <ul className="field-group">
+                <MultiInputContainer
```

Comment:
When we use inputs and handle submit, we do this in the context of a form. For example [agencies form](https://github.com/FromDoppler/doppler-webapp/blob/master/src/components/DopplerPlus/AgenciesForm.js) or you can take a look at login component or signup. To handle forms we use formik, and we have a pre defined way to work with formik. You can see all form helpers [here](https://github.com/FromDoppler/doppler-webapp/blob/master/src/components/form-helpers/form-helpers.js) Also this type of form you are using is being worked in another team, @ramniel is on it, perhaps you can wait for this to be ready.